screens/SearchingRequest.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
-import { StyleSheet, Text, View, Dimensions, Image, ActivityIndicator} from 'react-native';
+import { StyleSheet, Text, View, Dimensions, ActivityIndicator} from 'react-native';
+import Toolbar from './../components/Toolbar';
 
 export default class SearchingRequest extends React.Component {
   render() {
     return (
         <View style={styles.container}>
           <View style={styles.peachBox}>
-            <View style={styles.formatInPeachBox}>
-              <Image source ={require('./../assets/images/shopping-bag.png')} style={styles.bag}/>
-              <Image source ={require('./../assets/images/personicon.png')} style={styles.person}/> 
-            </View>
-            <View style={styles.whiteBox}>
-              <Text style={styles.header}>Searching for Request</Text>
-              <ActivityIndicator size='large' color="#605DF1" style={styles.loading}/>
+            <Toolbar marginBottom={0} marginTop={Dimensions.get('screen').height*.03}/>
```

Comment:
The toolbar will have the same space between itself and the top of the phone on every screen. That means that you don't have to pass in the `marginTop` prop since in theory it won't be changing from page to page